src/app/trips/trip-overview/trip-overview.component.html

```diff
@@ -0,0 +1,26 @@
+<div class="container">
+
+<h1>{{trip.title}}</h1>
+
+<span class="font700">
+  {{trip?.startDate?.toLocaleDateString('en-us', { month:"short", day:"numeric"})}}
+  &mdash;
+  {{trip?.endDate?.toLocaleDateString('en-us', { year:"numeric", month:"short", day:"numeric"})}}
+</span>
+
+<div class="">
+  Destination<span *ngIf="tripService.multipleLocations(trip.locations)">s</span>:
+  <br />
```

Comment:
let's remove any reference of br and replace it with CSS to move items where you need them.